libnetwork/drivers/overlay/encryption.go

```diff
@@ -629,7 +629,7 @@ func clearEncryptionStates() {
 	}
 	for _, sp := range spList {
 		if sp.Mark != nil && sp.Mark.Value == spMark.Value {
-			if err := nlh.XfrmPolicyDel(&sp); err != nil {
+			if err := nlh.XfrmPolicyDel(&sp); err != nil { // nolint:gosec
```

Comment:
Same here; this may be legit; should we capture `sp` inside the loop here? (`sp := sp`)